src/hotspot/share/cds/archiveHeapWriter.hpp

```diff
@@ -119,12 +128,6 @@ class ArchiveHeapWriter : AllStatic {
     int _field_offset;
   };
 
-  // The minimum region size of all collectors that are supported by CDS in
-  // ArchiveHeapLoader::can_map() mode. Currently only G1 is supported. G1's region size
-  // depends on -Xmx, but can never be smaller than 1 * M.
-  // (TODO: Perhaps change to 256K to be compatible with Shenandoah)
-  static constexpr int MIN_GC_REGION_ALIGNMENT = 1 * M;
```

Comment:
Couldn't you just move up the constant to the public section?
Also, I'm not sure what's the point of it being constexpr (rather that just const), but that is pre-existing.

Reply:
Yeah, I had problems with that. But I think I just misplaced the constant the last time around. Let me see what Windows GHA runs have to say about the code in new commit.

Reply:
Ah, now I remember the real reason I did this: I needed the constant out of the `#if INCLUDE_CDS_JAVA_HEAP` block. `INCLUDE_CDS_JAVA_HEAP` is only defined for `#if INCLUDE_CDS && INCLUDE_G1GC && defined(_LP64)`. So it breaks 32-bit builds. This will become moot after we remove Shenandoah check for region alignment, so I just worked it around in Shenandoah code now.